maintainers/maintainer-list.nix

```diff
@@ -1124,6 +1124,12 @@
     email = "sivaraman.balaji@gmail.com";
     name = "Balaji Sivaraman";
   };
+  balodja = {
+    email = "balodja@gmail.com";
+    github = "balodja";
+    githubId = 294444;
+    name = "Vladimir Korolev";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add balodja
```

Reply:
Done!